weaver/core/drivers/fabric-driver/server/helpers/helpers.ts

```diff
@@ -0,0 +1,764 @@
+/*
```

Comment:
I'd suggest to not use these helpers from fabric-cli for final version, try to find a similar function already existing in fabric-driver code, or if not present create new without depending upon any of these functions.

PS: This is not urgently needed to be done, it can be done when PR is about to be ready, but as we add more code the amount of refactoring needed might increase. So I'll leave upto you @outsidethecode when you want to make this change.

Reply:
Thanks @sandeepnRES, I think there should be a refactoring task for that which can be done anyway as part of the next version of the satp implementation.

Reply:
@outsidethecode Leaving it here wouldn't be appropriate as it duplicates code found elsewhere in the repository. This will raise questions about code quality and will be very hard to maintain going forward. It's OK to have this in test code but not in the project's main branch.

As @sandeepnRES says, the boilerplate code for network and chaincode operations already lies in the fabric-driver folder and can be reused. You should import just the missing parts from fabric-cli to here. I think this should not be more than a day of work.